config/cache_store.rb

```diff
@@ -1,8 +1,8 @@
 module Panoptes
-  module ElastiCache
+  module Cache
     def self.client
-      if cluster_name = ENV["ELASTIC_CACHE"]
-        Dalli::ElastiCache.new("#{cluster_name}:11211")
+      if cluster_name = ENV["CACHE_SERVER"]
+        Dalli::Client.new("#{cluster_name}:11211")
```

Comment:
Remove this `self.client` method and change this to use `ENV["MEMCACHE_SERVERS"]` 
https://github.com/petergoldstein/dalli#usage-with-rails-3x-and-4x
> If your servers are specified in ENV["MEMCACHE_SERVERS"] (e.g. on Heroku when using a third-party hosted addon), simply provide nil for the servers: